source/include/stdint.readme

```diff
@@ -25,13 +25,17 @@ typedef unsigned long        uint32_t;
 typedef long long            int64_t;
 typedef unsigned long long   uint64_t;
 
-#define INT8_MAX      ( ( signed char ) 127 )
-#define UINT8_MAX     ( ( unsigned char ) 255 )
-#define INT16_MAX     ( ( short ) 32767 )
-#define UINT16_MAX    ( ( unsigned short ) 65535 )
-#define INT32_MAX     2147483647L
-#define UINT32_MAX    4294967295UL
-#define INT64_MAX     9223372036854775807LL
-#define UINT64_MAX    18446744073709551615ULL
+#define INT8_MAX        ( ( signed char ) 127 )
+#define UINT8_MAX       ( ( unsigned char ) 255 )
+#define INT16_MAX       ( ( short ) 32767 )
+#define UINT16_MAX      ( ( unsigned short ) 65535 )
+#define INT32_MAX       2147483647L
+#define UINT32_MAX      4294967295UL
+#define INT64_MAX       9223372036854775807LL
+#define UINT64_MAX      18446744073709551615ULL
+
+#ifndef SIZE_MAX
```

Comment:
Is the `#ifndef` required considering that this file will be used on platforms that don't have `stdint.h` provided by the toolchain?

Reply:
The user could wish to define SIZE_MAX to a smaller value.

Reply:
A comment for this rationale in the file will be helpful 🙂

Reply:
I think the meaning of `#ifndef` is self-explanatory.